src/app/pages/preferences/page/forms/general-preferences-form.component.ts

```diff
@@ -45,32 +45,32 @@ export class GeneralPreferencesFormComponent implements OnInit, OnChanges, OnDes
         label:true,
         width:'300px',
         config:[
-          { 
-            type: 'select', 
-            name: 'userTheme', 
+          {
+            type: 'select',
+            name: 'userTheme',
             width:'300px',
-            placeholder: 'Choose Theme', 
+            placeholder: 'Choose Theme',
             options: this.themeOptions,
             value:this.themeService.activeTheme,
-            tooltip:'Pick your preferred theme.',
+            tooltip:'Choose a preferred theme.',
             class:'inline'
           },
-          { 
-            type: 'radio', 
-            name: 'metaphor', 
+          {
+            type: 'radio',
+            name: 'metaphor',
             width:'300px',
             placeholder: 'View Type Preference',
             options:[{label:'Cards',value:'cards'},{label:'Tables',value:'tables'},{label:'Auto',value:'auto'}],
             value:'cards',
-            tooltip: 'Choose your preferred view type.',
+            tooltip: 'Choose the preferred view type.',
           },
-          { 
-            type: 'checkbox', 
-            name: 'showTooltips', 
+          {
+            type: 'checkbox',
+            name: 'showTooltips',
             width: '300px',
-            placeholder: 'Enable Helpful Tooltips in Forms',
+            placeholder: 'Enable Help Text in Forms',
             value:this.prefs.preferences.showTooltips,
-            tooltip: 'This option enables/disables tooltips in forms.',
+            tooltip: 'Display Help Text icons with text in forms.',
```

Comment:
Not sure what "icons with text" means here.  Maybe just:
"Display help icons in forms."?